dlt-connector/src/proto/3_3/GroupFriendsUpdate.ts

```diff
@@ -0,0 +1,15 @@
+import { Field, Message } from '@apollo/protobufjs'
+
+// connect group together
+// only CrossGroupType CROSS (in TransactionBody)
+// https://www.npmjs.com/package/@apollo/protobufjs
+// eslint-disable-next-line no-use-before-define
+export class GroupFriendsUpdate extends Message<GroupFriendsUpdate> {
+  // if set to true, colors of this both groups are trait as the same
+  // on creation user get coins still in there color
+  // on transfer into another group which a connection exist,
+  // coins will be automatic swapped into user group color coin
+  // (if fusion between src coin and dst coin is enabled)
```

Comment:
the current vision for _colored coins_ base on the communityUuid, which means a X-Com-Transaction will transfer for its tx-amount a list of sub-amount-values each associated with its own communityUuid where it was originally created.
An account will treat its balance with a list of sub_amount-values from their original sender communities. GDDs from the same community are added to one amount value. Theer will exist further strategies to treat with the colored coins, especially for blacklisted ones